converter/api.py

```diff
@@ -0,0 +1,20 @@
+from shieldhit.parser import Parser, DummmyParser, ShieldhitParser
+
+
+def get_parser_from_str(parser_type: str):
+    """Get a converter object based on the provided type."""
+    # This is temporary, suggestions on how to do this better appreciated.
+    if parser_type.lower() == 'dummy':
+        return DummmyParser()
+    elif parser_type.lower() == 'shieldhit':
+        return ShieldhitParser()
+    else:
+        print(f"Invalid parser type \"{parser_type}\".")
+        raise
+
+
+def run_parser(parser: Parser, input_data: dict, output_dir: str) -> None:
```

Comment:
`Parser` should be made general and moved out of `shieldhit` module.

Reply:
I've created a separate file in the converter module named `common` it will contain all classes that are shared between parsers.

Reply:
good